src/Microsoft.Health.Dicom.SchemaManager/CommandLineOptions.cs

```diff
@@ -14,7 +14,9 @@ public class CommandLineOptions
 
     public string? ConnectionString { get; set; }
 
+    [Obsolete("Use Connection String instead for different authentication types.")]
     public SqlServerAuthenticationType? AuthenticationType { get; set; }
 
+    [Obsolete("Use Connection String instead for different authentication types.")]
     public string? ManagedIdentityClientId { get; set; }
```

Comment:
Once we fully migrate over to the use of `ConnectionString` only I'll remove these old fields